libs/blocks/preflight/panels/general.js

```diff
@@ -72,8 +72,12 @@ function findLinks(selector) {
   return [...document.body.querySelectorAll(selector)]
     .reduce((links, el) => {
       const url = getUrl(el);
-      if (!hrefs.includes(url.href)) {
-        hrefs.push(url.href);
+      const baseUrl = url.origin + url.pathname;
+      if (url.pathname.includes('/tools/caas')) {
+        return links;
+      }
+      if (!hrefs.includes(baseUrl)) {
+        hrefs.push(baseUrl);
```

Comment:
This could be a use-case for a `Set`. Maybe you can look into it and see if it makes sense to use it instead of checking in an array.